server/ctrlsubsonic/handlers_by_folder.go

```diff
@@ -80,7 +80,7 @@ func (c *Controller) ServeGetMusicDirectory(r *http.Request) *spec.Response {
 		Where("parent_id=?", id.Value).
 		Preload("AlbumStar", "user_id=?", user.ID).
 		Preload("AlbumRating", "user_id=?", user.ID).
-		Order("albums.right_path COLLATE NOCASE").
+		Order("albums.right_path").
```

Comment:
the `COLLATE NOCASE` IIRC was there to case insensitively sort albums. is there a way we could do it that works for both postgres and sqlite? not a big deal if not i suppose

Reply:
I have been trying to find a way - and it seems there is no good overlap in the SQL dialects around handling collation. Happy to dig further, but not optimistic.